### ISSUE TITLE
[CM Birds P19982] Picoides borealis - locality/spelling

### ISSUE THREAD

**Issue by KenBlankenship** [cm-vertnet/cm-birds#20]
This locality should be spelled/located as:

Broro Neck, McIntosh Co.	31.559414	-81.327297


.....................................................

You can [view the original detail page](http://portal.vertnet.org/o/cm/birds?id=p19982) on VertNet. Here are the original record contents:
<table>
        <thead>
          <tr>
            <th>Term</th>
            <th>Value</th>
          </tr>
        </thead>
        <tbody>
        
                  
          <tr>
            <td>Modified</td>
            <td id="Modified">
              
                2014-09-08
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Language</td>
            <td id="Language">
              
                en
              
            </td>
          </tr>
          
        
        
        
        
        
                  
          <tr>
            <td>InstitutionID</td>
            <td id="InstitutionID">
              
                urn:lsid:biocol.org:col:34702
              
            </td>
          </tr>
          
        
        
                  
          <tr>
            <td>InstitutionCode</td>
            <td id="InstitutionCode">
              
                CM
              
            </td>
          </tr>
          
                  
          <tr>
            <td>CollectionCode</td>
            <td id="CollectionCode">
              
                Birds
              
            </td>
          </tr>
          
        
        
                  
          <tr>
            <td>BasisOfRecord</td>
            <td id="BasisOfRecord">
              
                PreservedSpecimen
              
            </td>
          </tr>
          
        
        
        
                  
          <tr>
            <td>OccurrenceID</td>
            <td id="OccurrenceID">
              
                urn:catalog:CM:Birds:P19982
              
            </td>
          </tr>
          
                  
          <tr>
            <td>CatalogNumber</td>
            <td id="CatalogNumber">
              
                P19982
              
            </td>
          </tr>
          
                  
          <tr>
            <td>OccurrenceRemarks</td>
            <td id="OccurrenceRemarks">
              
                SOURCE: A.W.ANTHONY.
              
            </td>
          </tr>
          
        
                  
          <tr>
            <td>RecordedBy</td>
            <td id="RecordedBy">
              
                WORTHINGTON, W.W.
              
            </td>
          </tr>
          
        
                  
          <tr>
            <td>IndividualCount</td>
            <td id="IndividualCount">
              
                1
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Sex</td>
            <td id="Sex">
              
                female
              
            </td>
          </tr>
          
        
                  
          <tr>
            <td>ReproductiveCondition</td>
            <td id="ReproductiveCondition">
              
                HOW SEXED: UNMARKED MEANS.
              
            </td>
          </tr>
          
        
                  
          <tr>
            <td>EstablishmentMeans</td>
            <td id="EstablishmentMeans">
              
                native
              
            </td>
          </tr>
          
                  
          <tr>
            <td>OccurrenceStatus</td>
            <td id="OccurrenceStatus">
              
                present
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Preparations</td>
            <td id="Preparations">
              
                study skin
              
            </td>
          </tr>
          
        
        
        
        
        
        
        
        
        
        
        
                  
          <tr>
            <td>EventDate</td>
            <td id="EventDate">
              
                1889-01-25
              
            </td>
          </tr>
          
        
                  
          <tr>
            <td>StartDayOfYear</td>
            <td id="StartDayOfYear">
              
                25
              
            </td>
          </tr>
          
                  
          <tr>
            <td>EndDayOfYear</td>
            <td id="EndDayOfYear">
              
                25
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Year</td>
            <td id="Year">
              
                1889
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Month</td>
            <td id="Month">
              
                1
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Day</td>
            <td id="Day">
              
                25
              
            </td>
          </tr>
          
        
        
                  
          <tr>
            <td>FieldNumber</td>
            <td id="FieldNumber">
              
                AWA 0085
              
            </td>
          </tr>
          
        
        
        
        
                  
          <tr>
            <td>HigherGeography</td>
            <td id="HigherGeography">
              
                NORTH AMERICA;USA;GEORGIA;;;;;
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Continent</td>
            <td id="Continent">
              
                North America
              
            </td>
          </tr>
          
        
        
        
                  
          <tr>
            <td>Country</td>
            <td id="Country">
              
                United States
              
            </td>
          </tr>
          
                  
          <tr>
            <td>CountryCode</td>
            <td id="CountryCode">
              
                US
              
            </td>
          </tr>
          
                  
          <tr>
            <td>StateProvince</td>
            <td id="StateProvince">
              
                Georgia
              
            </td>
          </tr>
          
        
        
                  
          <tr>
            <td>Locality</td>
            <td id="Locality">
              
                BROW NECK
              
            </td>
          </tr>
          
                  
          <tr>
            <td>VerbatimLocality</td>
            <td id="VerbatimLocality">
              
                USA; GEORGIA; ; BROW NECK
              
            </td>
          </tr>
          
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
                  
          <tr>
            <td>ScientificName</td>
            <td id="ScientificName">
              
                Picoides borealis
              
            </td>
          </tr>
          
        
        
        
        
        
        
                  
          <tr>
            <td>HigherClassification</td>
            <td id="HigherClassification">
              
                Animalia;Chordata;Aves;Piciformes;Picidae;Picoides
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Kingdom</td>
            <td id="Kingdom">
              
                Animalia
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Phylum</td>
            <td id="Phylum">
              
                Chordata
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Class</td>
            <td id="Class">
              
                Aves
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Order</td>
            <td id="Order">
              
                Piciformes
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Family</td>
            <td id="Family">
              
                Picidae
              
            </td>
          </tr>
          
                  
          <tr>
            <td>Genus</td>
            <td id="Genus">
              
                Picoides
              
            </td>
          </tr>
          
        
                  
          <tr>
            <td>SpecificEpithet</td>
            <td id="SpecificEpithet">
              
                borealis
              
            </td>
          </tr>
          
        
                  
          <tr>
            <td>TaxonRank</td>
            <td id="TaxonRank">
              
                species
              
            </td>
          </tr>
          
        
        
        
                  
          <tr>
            <td>NomenclaturalCode</td>
            <td id="NomenclaturalCode">
              
                ICZN
              
            </td>
          </tr>
          
        
        
        
        
        </tbody>
      </table>

**Comment by StephenRogers** [comment]
You submitted the information that Brow Neck was incorrect and Broro Neck was the correct spelling on June 24 for another specimens. We had 9 different specimens from this locality and I corrected all at that time. The database is corrected for all 9 but will not show until I update to VertNet sometime in the future.

**Comment by KenBlankenship** [comment]
Hi, Stephen.

Sorry to repeat this... I am not keeping a tally of the various institutions and cat numbers I have submitted correx for via VertNet. I am compiling a master spreadsheet of all specimens taken in GA that are in VertNet and in several collections in the Southeast that are not yet online; it's around 15,000 at the moment so I forget sometimes if I've plugged in a correction for a locality at specific institutions.

Thx,
KB


----- Original Message -----
